### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by mrksbrg**
ESI Pro-SiVIC data generation representing the SMIRK ODD. Tested on three different machines.

### CODE REVIEW

File: pedestrian-generator/poetry.lock [hunk 1]
```diff
@@ -0,0 +1,424 @@
+[[package]]
+name = "appdirs"
```
Comment (Tomaszewski-Piotr) [inline pedestrian-generator/poetry.lock:2]: should we really check in the lock files?

**Review by Tomaszewski-Piotr — COMMENTED** [review]
should we really have lock files under version control?

File: pedestrian-generator/poetry.lock (hunk 1 above)
Comment (updbqn) [reply pedestrian-generator/poetry.lock:2]: The lockfile is there to ensures that everyone has exactly the same version of all dependencies. See e.g. https://python-poetry.org/docs/basic-usage/#commit-your-poetrylock-file-to-version-control.

**Comment by Tomaszewski-Piotr** [comment]
aaa ok, roger that. ignore the comment then!

**Pull request merged**